### PR TITLE
Add optional to all preferred first name fields

### PR DESCRIPTION
**Pull request by samaratrilling**
# Previously, on NoRent:
Screen Shot 2021-07-09 at 11 27 46 AM

# This time, on NoRent:
Screen Shot 2021-07-09 at 11 27 07 AM

# Previously, on justfix.nyc:
Screen Shot 2021-07-09 at 11 32 15 AM

# This time, on justfix.nyc:
Screen Shot 2021-07-09 at 11 32 00 AM

### CODE REVIEW

**Comment by samaratrilling** [comment]
I'm open to the idea that we shouldn't have a redundant 'optional' label on Preferred Name in Account Settings. What do you think?

File: users/migrations/0011_justfixuser_preferred_first_name.py [hunk 1]
```diff
@@ -6,13 +6,18 @@
...
-            name='preferred_first_name',
-            field=models.CharField(blank=True, help_text='The first name Justfix will call the user by. Optional.  May be different from their legal first name.', max_length=150, verbose_name='Preferred first name'),
+            model_name="justfixuser",
+            name="preferred_first_name",
+            field=models.CharField(
+                blank=True,
+                help_text="The first name Justfix will call the user by. Optional.  May be different from their legal first name.",
+                max_length=150,
+                verbose_name="Preferred first name",
+            ),
```
Comment (toolness) [inline users/migrations/0011_justfixuser_preferred_first_name.py:21]: I was confused by this at first, but I'm guessing what happened is that you saved this file from VSCode, which immediately ran Black on it.  Normally we don't check for Black formatting on these files b/c they're auto-generated by Django, but it's hard/annoying to also tell VSCode to ignore them when auto-formatting on save.

Anyways as far as I can tell this is purely Black at work, and contains no semantic changes, but let me know if I'm mistaken!

**Review by toolness — APPROVED** [review]
Nice, this looks great, thanks!

File: users/migrations/0011_justfixuser_preferred_first_name.py (hunk 1 above)
Comment (samaratrilling) [reply users/migrations/0011_justfixuser_preferred_first_name.py]: Ah yes indeed, this was Black at work. I'll revert - I think it causes more confusion than it's worth.

**Comment by toolness** [comment]
> I'm open to the idea that we shouldn't have a redundant 'optional' label on Preferred Name in Account Settings. What do you think?

Oh hmmmm good question.  I think maybe it's okay to leave it there, just so folks aren't confused?  It's also potentially a bit unclear what exactly will happen if the user leaves it blank... Hmm. For the descriptive text "The name you'd like people to call you by", what do you think of changing it to this:

> The name you'd like people to call you by. If you leave this blank, we'll just call you by your legal first name (Boop).

That way the consequences of what happens if the user leaves the field blank will never be unclear.  But maybe it's already super clear, I dunno.

Anyways I think that either way, we should just merge what you have now but consider either of these changes for later!

**Pull request merged**